effects/gh-pages/README.md

```diff
@@ -0,0 +1,83 @@
+# `gh-pages` effect
```

Comment:
Please read the "top level" review message before diving into the file comments. (The one in the conversation tab)